test/functional/action-params.spec.ts

```diff
@@ -72,6 +78,46 @@ describe("action parameters", () => {
         // reset metadata args storage
         defaultMetadataArgsStorage().reset();
 
+
+        @Middleware()
+        class SessionMiddleware implements MiddlewareInterface {
```

Comment:
I have tried.
* Middleware define in `before` function inside ✅
* Middleware define in other file
    - file head `import`  ❌
    - `require` ✅
    - delete `defaultMetadataArgsStorage().reset()` ✅